CHANGELOG.adoc

```diff
@@ -64,6 +65,7 @@ As a result, the following maven modules have been deleted: `sirius-web-sample-a
 - [releng] Add a dependency to CycloneDX to compute the software bill of materials during the build
 - https://github.com/eclipse-sirius/sirius-web/issues/3437[#3437] [core] Migrate frontend to MUI 5
 - https://github.com/eclipse-sirius/sirius-web/issues/3392[#3392] [diagram] Add a dependency to `pathfinding`
+- https://github.com/eclipse-sirius/sirius-web/issues/3846[#3846] [core] Migrate frontend to React 18
```

Comment:
I'll move those details to the CHANGELOG of 2024.11.0 and add additional details on react-dom, react-router, etc